Cargo.toml

```diff
@@ -14,10 +14,10 @@ clap = { version = "4.5.6", features = ["derive", "env"] }
 dashmap = "6.1.0"
 derivative = "2.2.0"
 derive_more = { version = "1.0.0", features = ["full"] }
-ethcontract = { version = "0.25.8", default-features = false, features = ["aws-kms"] }
+ethcontract = { git = "https://github.com/cowprotocol/ethcontract-rs", rev = "c9ec7c090ea649f1cd2a999caae71aa571251b48", default-features = false, features = ["aws-kms"] }
```

Comment:
Are the commits here still needed? Do you expect many more changes needed in ethcontract-rs to aid the alloy migration?

Reply:
If doing new releases turns out to be a problem let's stick with pinning commits. After all we want to migrate away from that library anyway. 👌